lib/takeaway.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'menu'
+require_relative 'sms'
+
+class Takeaway
+  attr_reader :customer_order, :total, :sms
+  @menu = []
+
+  def initialize(menu = Menu.new, sms_obj)
+    @customer_order = []
+    @menu = menu
+    @total = 0.0
+    @sms = sms_obj || Sms.new
+  end
+
+  def show_menu
+    @menu.show_menu
+  end
+
```

Comment:
I think there are one or two places where you create instance vars, for things which could be local vars.

For example, @customer_order could be a local var, I think, since you pass it into the `#verify_order` method as an argument.

Do you know how local variables and instance variables behave differently?